colabs/artifact_basics/Artifact_Basics.ipynb

```diff
@@ -0,0 +1,355 @@
+{
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "gsp5VlKQVMud"
+      },
+      "source": [
+        "<a href=\"https://colab.research.google.com/github/wandb/examples/blob/master/colabs/pytorch/Artifacts_Basics.ipynb\" target=\"_parent\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/></a>\n",
+        "<!--- @wandbcode{artifacts-basics} -->"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "TbzoV1XvVMug"
+      },
+      "source": [
+        "<img src=\"http://wandb.me/logo-im-png\" width=\"400\" alt=\"Weights & Biases\" />\n",
+        "\n",
+        "<!--- @wandbcode{pytorch-video} -->\n"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "MGA0PLfsVMuh"
+      },
+      "source": [
+        "\n",
```

Comment:
Suggest changing 

Use [Weights & Biases](https://wandb.com/) for machine learning experiment tracking, dataset versioning, and project collaboration.

to something like

Use [Weights & Biases](https://wandb.com/) for machine learning experiment tracking, dataset and model versioning and management, collaboration and more.

in order to highlight benefits of W&B for artifacts up front.